modules/machineset-gcp-shielded-vms.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+
+ifeval::["{context}" == "cpmso-using"]
+:cpmso:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="machineset-gcp-shielded-vms_{context}"]
+= Configuring Shielded VM options by using machine sets
+
+By editing the machine set YAML file, you can configure the Shielded VM options that a machine set uses for machines that it deploys.
+
+For more information about Shielded VM features and functionality, see the GCP Compute Engine documentation about link:https://cloud.google.com/compute/shielded-vm/docs/shielded-vm[Shielded VM].
+
+.Procedure
+
+. In a text editor, open the YAML file for an existing machine set or create a new one.
+
+. Edit the following section under the `providerSpec` field:
++
+[source,yaml]
+----
+ifndef::cpmso[]
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+endif::cpmso[]
+ifdef::cpmso[]
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+endif::cpmso[]
+...
+spec:
+  template:
+    spec:
+      providerSpec:
+        value:
+          shieldedInstanceConfig: <1>
+            integrityMonitoring: Enabled <2>
+            secureBoot: Disabled <3>
+            virtualizedTrustedPlatformModule: Enabled <4>
+...
+----
+<1> In this section, specify any Shielded VM options that you want.
+<2> Specify whether UEFI Secure Boot is enabled. Valid values are `Disabled` or `Enabled`.
```

Comment:
If you wanted to get rid of the alliteration in `valid values`, I _think_ you substitute for `accepted`.